src/components/PokemonCard/PokemonCard.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import axios from "axios";
+import "./PokemonCard.css"
+
+function PokemonCard({ endpoint }) {
+    const [ pokemon, setPokemon ] = useState({})
```

Comment:
Als je hier zou kiezen voor een default waarde `undefined` dan kan je check verderop een stuk simpeler
```suggestion
    const [ pokemon, setPokemon ] = useState();
```